### PR TITLE
Added batch normalization to cnn_training

### PR DESCRIPTION
**Pull request by aditya-rajagopal**
Batch normalization at every convo layer. 
Needs to be updated after finding a good model
Currently the base network does not train.
Needs to be tuned and needs to be able to reduce the over-fitting brought observed in the diffparse branch

### CODE REVIEW

**Comment by aditya-rajagopal** [comment]
Corrected all bugs. Batch normalization increased validation accuracy by 5%.
